jina/helper.py

```diff
@@ -1237,15 +1237,8 @@ def _get_ip(url):
         'https://checkip.amazonaws.com/',
     ]
 
-    threads = []
-
     for idx, ip in enumerate(ip_server_list):
-        t = threading.Thread(target=_get_ip, args=(ip,))
-        threads.append(t)
-        t.start()
-
-    for t in threads:
-        t.join(timeout)
+        results.append(_get_ip(ip))
 
     for r in results:
         if r:
```

Comment:
this can be removed if we apply the suggestion above

Reply:
Suggestion applied, good catch!